### PR TITLE
Sync internal changes

### PR DESCRIPTION
**Pull request by njhill**

### CODE REVIEW

File: src/main/java/com/ibm/watson/modelmesh/LogRequestHeaders.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
+/*
+ * *****************************************************************
+ *
```
Comment (pvaneck) [inline src/main/java/com/ibm/watson/modelmesh/LogRequestHeaders.java:3]: Should be replaced with Apache header.

File: src/test/java/com/ibm/watson/modelmesh/ModelMeshHeaderLoggingTest.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
+/* ***************************************************************** */
+/*                                                                   */
+/* IBM Confidential                                                  */
+/* OCO Source Materials                                              */
+/*                                                                   */
+/* (C) Copyright IBM Corp. 2001, 2021                                */
+/*                                                                   */
```
Comment (pvaneck) [inline src/test/java/com/ibm/watson/modelmesh/ModelMeshHeaderLoggingTest.java:7]: Same as above.

Comment (njhill) [reply src/test/java/com/ibm/watson/modelmesh/ModelMeshHeaderLoggingTest.java:7]: @pvaneck thanks! knew there was something I was forgetting

**Comment by njhill** [comment]
Thanks @pvaneck, hopefully this should be good to rebase-and-merge now.

**Review by pvaneck — APPROVED** [review]
Thanks, @njhill. Looks good. Feel free to rebase and merge.

**Pull request merged**